src/EnergyPlus/CurveManager.cc

```diff
@@ -824,6 +826,7 @@ namespace Curve {
                                                                      _,
                                                                      state.dataIPShortCut->cAlphaFieldNames,
                                                                      state.dataIPShortCut->cNumericFieldNames);
+            state.dataInputProcessing->inputProcessor->unusedInputs.emplace(CurrentModuleObject, Alphas(1));
             ++CurveNum;
             GlobalNames::VerifyUniqueInterObjectName(state,
```

Comment:
This doesn't have to be part of this PR, but look at how I am using `std::map` to implement both `VerifyUniqueObjectName` and `GetObjectIndex` in one fast structure. There are examples in `Fans`, `OutputProcessor` and `SetPointManager`.